gr-fec/python/fec/polar/channel_construction.py

```diff
@@ -15,7 +15,7 @@
 from .channel_construction_bec import calculate_bec_channel_capacities
 from .channel_construction_bec import design_snr_to_bec_eta
 from .channel_construction_bec import bhattacharyya_bounds
-from .helper_functions import *
+from .helper_functions import np
```

Comment:
I suggest the very common
```
import numpy as np
```
rather than importing `np` from `helper_functions` (which contains an `import numpy as np`).

This remark also applies to `gr-fec/python/fec/polar/testbed.py`